docs/pages/guides/using-sentry.mdx

```diff
@@ -294,6 +294,16 @@ In order to ensure that errors are reported reliably, Sentry defers reporting th
 
 Unless `enableInExpoDevelopment: true` is set, all your dev/local errors will be ignored and only app releases will report errors to Sentry. You can call methods like `Sentry.Native.captureException(new Error('Oops!'))` but these methods will be no-op.
 
+## Troubleshooting
+
+<Collapsible summary="I'm seeing `error: project not found` in my build logs">
```

Comment:
```suggestion
<Collapsible summary={<>I'm seeing <CODE>error: project not found</CODE> in my build logs</>}>
```

Correction for the collapsible title formatting, you might also need to add import for `CODE` element, if it is not present.

Reply:
Ah great, I didn't know you could use it like that. Thanks! Updated.